doc/content/getstarted.md

```diff
@@ -2,62 +2,82 @@
 title: Get Started
 ---
 
-## The Scientific Python theme for Hugo
-
 The **Scientific Python Hugo Theme** is a theme for the
 [Hugo](https://gohugo.io) static site generator based on the
-[Fresh](https://github.com/StefMa/hugo-fresh) theme.
+[PyData Sphinx Theme](https://pydata-sphinx-theme.readthedocs.io/en/latest/).
+
+To use this theme on your site, follow these instructions.
+
+## Install Hugo
 
-To use the theme, you will need to
-[download hugo](https://github.com/gohugoio/hugo/releases)
-and place it on your path.
+Hugo is the program that reads the site's source files and generates the HTML and other files that are served to visitors in their Web browsers.
 
-## Download and install
+1. Install Hugo according to [its installation instructions](https://gohugo.io/installation/).  The standard edition of Hugo is sufficient; you may install the extended edition, but it is not required by this theme.
 
-1. Download [the theme ZIP file](https://github.com/scientific-python/scientific-python-hugo-theme/archive/refs/heads/main.zip) and extract it.
+2. Verify that the `hugo` binary is present in your `PATH` environment variable by running the command `hugo version`, which should output a message like:
+
+   ```
+   hugo v0.120.3-a4892a07b41b7b3f1f143140ee4ec0a9a5cf3970 linux/amd64 BuildDate=2023-11-01T17:57:00Z VendorInfo=gohugoio
+   ```
 
-2. Copy the `doc` folder as a template of your new website:
+## Install theme
+
+This theme is designed to be used as a Git submodule inside the repository that contains the Web site's source files, which Hugo builds from.
+
+1. Enter the directory of your site's Git repository.  Or if your site doesn't yet have a Git repository, initialize one in a new directory, e.g. by running the command:
 
    ```sh
-   cp -r scientific-python-hugo-theme-main/doc ./my-website
-   cd my-website
+   mkdir new-site && cd new-site && git init
    ```
 
-3. Initialize git and add `scientific-python-hugo-theme` as a submodule:
+2. Add the theme repository as a submodule:
 
    ```sh
-   git init
    git submodule add https://github.com/scientific-python/scientific-python-hugo-theme themes/scientific-python-hugo-theme
    ```
 
-4. Build your site:
+3. Copy the theme's documentation site as a template for your site:
 
    ```sh
-   make serve
+   cp -a themes/scientific-python-hugo-theme/doc/* .
    ```
 
-Browse to `http://localhost:1313`, and hopefully you will see your new site!
+4. Test the site by running `make serve`.  The command's output should include a line like:
 
-## Build HTML
+   ```
+   Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
+   ```
 
-Run `make html`. Output appears in `./public`.
+   Load that URL in your Web browser and you should see the the theme's documentation page, indicating that installation is successful.
```

Comment:
```suggestion
   Load that URL in your browser and you should see the the theme's documentation page, which you can now customize with your own content.
```

Reply:
Not to argue with you, but I think it might be good to leave some wording to the effect that this indicates a successful installation, as a way to confirm it to the user.  Otherwise, there is no kind of "Installation successful" message, as one might get when installing other software.

Reply:
There is no installation step, really, but perhaps you can say something like "indicating that the site compiled successfully".

Reply:
Updated in https://github.com/scientific-python/scientific-python-hugo-theme/pull/401/commits/c6d4f928d7d9c5ef39a228d626f8b9dff14aa063.  Please let me know if you'd like anything else changed.